manifests/config.pp

```diff
@@ -1,5 +1,8 @@
 # Configure dns
 class dns::config {
+  validate_array($dns::forwarders)
```

Comment:
In all our modules we have it in init.pp so it would be more consistent. It's also closer to where the users passes it in.